setup.py

```diff
@@ -3,8 +3,28 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
-with open("mplc/requirements.txt", "r") as fh:
-    requirements = fh.read()
+requirements = [
+    'h5py==2.10.0',
+    'joblib==1.0.0',
+    'librosa==0.8.0',
+    'matplotlib==3.3.3',
+    'numpy==1.19.4',
+    'scipy==1.5.4',
+    'scikit-learn==0.23.2',
+    'pandas==1.1.5',
+    'seaborn==0.11.0',
+    'loguru==0.5.3',
+    'tensorflow==2.4.1',
+    'ruamel.yaml==0.16.12',
+    'flake8==3.8.3',
+    'setuptools==50.3.0',
+    'wheel==0.35.1']
```

Comment:
I think `flake8`, `setuptools` and `wheel` are required respectively for linting and building, I am not sure that it needs to be included in the `setup.py`, but I am not sure about this

Reply:
I agree, as they are in dev-requirements they can be removed here. Thanks for noticing it.